platyPS.schema.md

```diff
@@ -41,17 +44,19 @@ It closely resembles output of `Get-Help`.
 
         // for every parameter
             // default value is non-mandatory
-            ### {Parameter name} [{Parameter type}] = {Parameter default value}
+            ### {Parameter name}
```

Comment:
Not sure I understand here why you removed `{Parameter default value}`. Is there somewhere else this information is represented?